### DISCUSSION TITLE
Releases Atom feed controls/filters

### DISCUSSION THREAD

**Discussion by mfulton26** [community/community#128589] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I like the idea of being able to get alerted on releases via Atom Feeds, but I find that for some repositories they have too much noise.

e.g. <https://github.com/remix-run/react-router/releases.atom> returns all the tags (almost the exact same payload as <https://github.com/remix-run/react-router/tags.atom>)

I want a way to get an Atom feed for only "latest" releases (no tags that aren't releases and no pre-releases). Is this already possible with some URL search parameters that I can't find the documentation for? If not, I'd like to see more controls/filters here to subscribe to only releases.

**Comment by WaterSwallow** [comment]
> ### Body
> I like the idea of being able to get alerted on releases via Atom Feeds, but I find that for some repositories they have too much noise.
> 
> e.g. https://github.com/remix-run/react-router/releases.atom returns all the tags (almost the exact same payload as https://github.com/remix-run/react-router/tags.atom)
> 
> I want a way to get an Atom feed for only "latest" releases (no tags that aren't releases and no pre-releases). Is this already possible with some URL search parameters that I can't find the documentation for? If not, I'd like to see more controls/filters here to subscribe to only releases.

Currently, GitHub does not provide a way to filter Atom feeds for "latest" releases specifically via URL parameters. The /releases.atom feed includes all releases and pre-releases.

An alternative may be using an RSS reader that can filter results based on specific criteria, though how to set such filters would be dependent on the application in question.

You could also consider reaching out in the GitHub community forums or submitting a feature request directly to GitHub to add more granular control over their release feeds. This isn't something that is user-controllable currently, but if it's something that enough people are interested in, they may consider adding it.

**Comment by LiteBrite82** [comment]
Hi @mfulton26 
Not that I'm aware of. I will ad the "enhancement" tag to the discussion so we can track community interest.

**Comment by nazar-pc** [comment]
https://github.com/riscv/riscv-arch-test has a lot of "pre-releases" that pollute the feed.
https://github.com/nextcloud/server also has a lot of beta/rc pre-releases that I have no interest in.

Many examples like this. It would be great to be able to exclude pre-releases or better yet specify exclusion filter for the returned feed.
